### ISSUE TITLE
JMT MCMT (jmt_mcmt) has failed to load correctly

### ISSUE THREAD

**Issue by Domatron** [jediminer543/JMT-MCMT#122]
I was running the mod pack “Better MC 1.16.5” and I removed roadrunner since it's not compatible. The server ran great for 8 hours, however after rebooting the server I have received this issue.

I am unsure how this has happened, and any feedback would be appreciated.
Thank you for your time!
[crash-2022-07-13_03.06.59-fml.txt](https://github.com/jediminer543/JMT-MCMT/files/9098699/crash-2022-07-13_03.06.59-fml.txt)

**Comment by quentin452** [comment]
Try to delete krypton reforged, and retry

And i have a question ,where you found jmt_mcmt 0.23.89-PRE  

on curseforge the latest is 22.87 in 1.16.5

**Comment by Dominic200527** [comment]
It's in the discord, dev artifacts. And I'll remove krypton reforged and let you know how it goes.

**Comment by Domatron** [comment]
I have deleted Krypton and no success in the server starting, I noticed this in the crash report which is kinda odd.
"Exception message: java.lang.NullPointerException: Registry Object not present: cavesandcliffs:axolotl_attackables"
^ I see this now, which is weird to me.
[crash-2022-07-13_14.33.10-fml.txt](https://github.com/jediminer543/JMT-MCMT/files/9103615/crash-2022-07-13_14.33.10-fml.txt)
(P.S Dominic200527 is my GitHub on my phone)

**Comment by Domatron** [comment]
Yes, I have used the stable version from curseforge. Thought of trying the pre version as a last resort to it.

**Comment by Nebula240** [comment]
I have the same issue with 
"Exception message: java.lang.NullPointerException: Registry Object not present: cavesandcliffs:axolotl_attackables"
